api/server/user/data.go

```diff
@@ -156,8 +156,8 @@ func (s *Service) WatchLogs(req *userPb.WatchLogsRequest, srv userPb.UserService
 	return nil
 }
 
-// CidInfo returns information about cids managed by the FFS instance.
-func (s *Service) CidInfo(ctx context.Context, req *userPb.CidInfoRequest) (*userPb.CidInfoResponse, error) {
+// CidSummary gives a summary of the storage and jobs state of the specified cid.
+func (s *Service) CidSummary(ctx context.Context, req *userPb.CidSummaryRequest) (*userPb.CidSummaryResponse, error) {
```

Comment:
This method is surprisingly slow, and I need to figure out why. When I call it against our powergate instance with about 700 cids stored, it takes 10s of seconds to return the data. The amount of data is small, so it must be some processing in here that is taking a while.

Reply:
I'm not sure when you test that, but today there were some heavy processes in the Atlas db.
Maybe you got unlucky? Mentioning it since was a special day today.
Dismiss if isn't the case... might be a real prob.

Reply:
Tested before the downtime. Will have to dig in and see what's up.